ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -1,8 +1,384 @@
+      netRegex: NetRegexes.startsUsing({ id: '560[DE]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 12,
+      response: Responses.getOut(),
+    },
+    {
+      id: 'E9S Empty Plane',
+      netRegex: NetRegexes.startsUsing({ id: '55EF', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+      run: (data) => {
+        data.phase = 'empty';
+      },
+    },
+    {
+      id: 'E9S Flood Of Emptiness',
+      netRegex: NetRegexes.startsUsing({ id: '55F0', source: 'Cloud Of Darkness', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Tile Positions',
+        },
+      },
+    },
+    {
+      id: 'E9S Curse Of Darkness',
+      netRegex: NetRegexes.gainsEffect({ effectId: '953' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_, matches) => matches.duration - 3,
+      alertText: (data, matches, output) => output.text({ player: data.ShortName(matches.target) }),
+      outputStrings: {
+        text: {
+          en: 'Point Face Laser Out',
+        },
+      },
+    },
+    {
+      id: 'E9S Hypercharged Condensation',
+      netRegex: NetRegexes.startsUsing({ id: '560C', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 2,
+      durationSeconds: 5,
+      response: Responses.killAdds(),
+    },
+    {
+      id: 'E9S The Art Of Darkness Right',
+      netRegex: NetRegexes.startsUsing({ id: '5A95', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 8,
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'E9S The Art Of Darkness Left',
+      netRegex: NetRegexes.startsUsing({ id: '5A96', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 8,
+      response: Responses.goRight(),
+    },
+    {
+      id: 'E9S Full-Perimeter Particle Beam',
+      netRegex: NetRegexes.startsUsing({ id: '5629', source: 'Cloud Of Darkness', capture: false }),
+      // Let Curse of Darkness trigger resolve first
+      delaySeconds: 2,
+      durationSeconds: 5,
+      response: Responses.getIn(),
+    },
+    {
+      id: 'E9S Deluge Of Darkness',
+      netRegex: NetRegexes.startsUsing({ id: '55F1', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe('alert'),
+      run: (data) => {
+        delete data.phase;
+      },
+    },
+    {
+      id: 'E9S The Third Art Of Darkness Right -> Protean -> Left',
+      netRegex: NetRegexes.startsUsing({ id: '5603', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 18,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Left -> Protean -> Right',
+        },
+      },
+    },
+    {
+      // TODO: Unconfirmed
```

Comment:
I am skeptical of this.  Can you get more data or remove for now? Mostly I don't want to give anybody an incorrect left/right call.